### DISCUSSION TITLE
alienFX-gui lost lighting control

### DISCUSSION THREAD

**Discussion by tinylvis** [T-Troll/alienfx-tools#445] [Q&A]
I was trying to turn off the alienhead but would not cooperate so I attempted to uninstall alienfx and reinstall it with the msi file but the program came up without the "lights" tab. I am unable to control any lighting on the keyboard, power button, or alienhead on the cover. The lighted alienhead of the laptop cover now stays on after shutdown...I'm stumped.

I have a 2019 Alienware M17 R1. I have been using alienfx-gui for years to replace AWCC which was horrible. I uninstalled AWCC and could never get it to reinstall again so I turned to alienFX.
![notebook-alienware-m-17-r1](https://github.com/user-attachments/assets/56877856-be6e-4399-8b2b-ed42047502a9)

**Comment by T-Troll** [comment]
Ughhh... Your light chip hangs. It can happen in rare situations.

To reset, shutdown (NOT sleep!) the system, remove ALL cables (including power cord), wait about 15 seconds (or until logo light is down), start back.

PS: To minimize probability of such hangs, do not change power button color frequently.

**Comment by tinylvis** [comment]
Thanks for the quick reply. 
I tried it a couple of times but no joy.

**Reply by tinylvis** [reply]
Sorry, Finally worked.
any idea on the missing "light" tab on the app?

**Reply by T-Troll** [reply]
It's normal if light chip hangs and can't be initialized.

**Reply by techlogik** [reply]
Ok. So this happened to me on an R12. The gui has no tabs for lighting. But now my power light is stuck in some random color and flashing quickly. Nothing works to make the gui return or fix the flashing light...power off etc..unplug blah...nothing. Now I got a flashing pink light for power and no way to fix it. Any ideas? I run alienfx-cli.exe and it just says "Dell API not found and Light devices not found...exiting!

**Reply by T-Troll** [reply]
Fast power button light blink means it was set as a common light.
In fact, 550/551 chip is tricky for working with hardware power button...
And yes, it just hangs and doesn't response from USB. For notebooks, it's enough to power off and remove ALL cables (including power cord) for about 15 seconds to restart it (clear configuration, in fact). For the desktops... If this not work, try to reset BIOS settings as well.

**Reply by techlogik** [reply]
Thanks for the reply last night. I ended up solving it by reinstalling the AWCC software. Apparently, SW Fusion got corrupt somehow and that is why the tabs wouldn't show. Now the issue I have is the lights are reversed. The power is the Ring, and Ring is Power. So when I select one and change the color the other changes.  Yeah......only way I can also make them both turn on is by opening the software and selecting the Power, and the ring lights up now. The Power button comes on the color I want from start. But ring won't. Thx

**Reply by techlogik** [reply]
Final update. I was finally able to get this working even though on the R12 the Power/Ring is reversed. When you change the Ring color, it changes the Power button, and vice versa. But I created two unique groups with each one and custom color and it seems to finally hold now across reboots, sleep mode/re-awake. Cool tool though. Love the Fan control. The AWCC Dell tool can't control anything finely on the fans, it is either going 20% or %50 nothing in between. Plus I have a custom fan/replaced some of my fans to my case and this app controls the fans/curve much better. So I have a good balance of reasonable noise and cooling/temps along with some custom cool lights now.  Awesome product!  Sorry to piggy back, but since this was the same issue I had figured I would add to it for others and some things to try. Like you said, R12 is tricky for sure. Newer models are likely much easier to control.

**Reply by T-Troll** [reply]
Thank you.
Feel free to share any ideas about how to make light configuration simpler, as well as other feedback.
